src/net_processing.cpp

```diff
@@ -4252,6 +4262,16 @@ void PeerManager::ProcessMessage(CNode& pfrom, const std::string& msg_type, CDat
             AnnounceTxs(remote_missing, pfrom, msgMaker, m_connman);
             m_connman.PushMessage(&pfrom, msgMaker.Make(NetMsgType::RECONCILDIFF, uint8_t(RECON_SUCCESS), local_missing));
             peer->m_recon_state->FinalizeReconciliation(true, Peer::ReconState::LocalQAction::Q_RECOMPUTE, local_missing.size(), remote_missing.size());
+        } else {
+            // Initial reconciliation failed.
+            // Store the received sketch and the local sketch, request extension.
+
+            assert(remote_sketch != nullptr);
```

Comment:
Is it possible for the peer to send us a message that causes `remote_sketch` to be nullptr here?

Reply:
Adding protection for this case.